tests/kernel/wave/speculative_decode_test.py

```diff
@@ -66,6 +69,10 @@ def tree_speculative_sampling_target_only(
 
     wave_kernel = get_wave_speculative_decoding_kernel(d)
     threshold_acc = max(threshold_acc, 1e-9)
+    cur_prob_offset_vec = torch.empty(
+        [batch_size], dtype=torch.int32, device=draft_probs.device
+    )
+    last_accepted_retrive_idx_vec = [None] * batch_size
```

Comment:
shouldn't this also be a torch.empty?

Reply:
this one is used for indexing so just regular list[int] is OK, but I can switch it out to torch.empty for uniformity

Reply:
done!